### ISSUE TITLE
ReferenceError: __default is undefined

### ISSUE THREAD

**Issue by hktr92** [denoland/deno#9307]
Similar with #8952 but behaves differently.

In Deno 1.6.1, when executing `deno bundle`, I had an error in console:

    deno run server.bundle.js
    error: Uncaught TypeError: Cannot set property 'placeholder' of undefined
    bind.placeholder = {
                     ^
        at file:///projects/hktr92/deno/oak-playground/server.bundle.js:3834:18

In Deno 1.7, however, it produces the following error:

    deno run server.bundle.js 
    error: Uncaught ReferenceError: __default is not defined
        return setToString(overRest(func, start, __default), func + '');
                                                 ^
        at baseRest (file:///projects/hktr92/deno-api-skel/server.bundle.js:315:46)
        at file:///projects/hktr92/deno-api-skel/server.bundle.js:856:12

which is caused by:
```js
function baseRest(func, start) {
    return setToString(overRest(func, start, __default), func + '');
}
```

the issue is caused by export syntax from lodash (see: https://github.com/lodash/lodash/blob/4.17.20-es/lodash.js)

```js
export { default as add } from './add.js';
// same with others
```

now, if I import them individually, it's not that a big of a problem, as I was following a udemy course for this, but it breaks things up.

**Comment by lucacasonato** [comment]
Please use Deno 1.7.0. We do not support old versions of Deno. There is a good chance this is fixed in either 1.7.0 (`deno upgrade`), or in the latest canary release (`deno upgrade --canary`).

**Comment by kdy1** [comment]
Can you share more code?
I'm not sure how can I make the test case.

**Comment by mengFxing** [comment]
![image](https://user-images.githubusercontent.com/19838247/108318034-22877f00-71fa-11eb-9434-30e3b25f70d6.png)

**Comment by czabaj** [comment]
Have the same problem with empty reference of `__default` although mine originates from [default export of `toObjMap` in adelsz/graphql-deno](https://github.com/adelsz/graphql-deno/search?q=toObjMap).

I offer my repo with a demo of the issue, in [czabaj/gql-refinery#issue-bundle](https://github.com/czabaj/gql-refinery/tree/issue-bundle)

If you checkout the repo, then do

1. `make bundle`
1. `make nodeBundleIssue` -- attempt to run the bundle in Node.js,
1. `make denoBundleIssue` -- attemts to run the bundle in Deno,

both attempts fail on

> ReferenceError: __default is not defined

**Comment by hktr92** [comment]
sorry for such a long time of no response. i had a busy month.

on my work laptop, i have manjaro with deno installed from aur. i didn't notice it was deno 1.6.1, so i upgraded to deno 1.7, which had the actual issue with `default` being renamed into `__default`. this happened before i opened this issue, back in january.

right now i'm currently on my windows machine and it seems that deno v1.7.5 works without any issues. only after i opened this issue i noticed it uses swc to handle es stuff, so it might be a bug in a version of swc used between deno 1.7.0 - 1.7.4 (i didnt checked every version of it to see exactly which one was, since from my point it's fixed).

i think that this issue can be closed for now, as the original scope of the issue isn't valid anymore from my point of view.

anyway, if it's needed, here's a simplistic example: https://github.com/hktr92/deno-default-bug
i've committed the bundled code too, just to have a snapshot of it.

sorry, again, for not responding in time.

**Comment by bartlomieju** [comment]
Fixed in 1.9.0
